ci/release/update-version.sh

```diff
@@ -49,3 +50,6 @@ for DEP in "${DEPENDENCIES[@]}"; do
   done
   sed_runner "/\"${DEP}==/ s/==.*\"/==${NEXT_SHORT_TAG_PEP440}.*,>=0.0.0a0\"/g" python/cucim/pyproject.toml
 done
+
+# Update CI image tags of the form {rapids_version}-{something}
+sed_runner "s/:[0-9]*\\.[0-9]*-/:${NEXT_SHORT_TAG}-/g" ./CONTRIBUTING.md
```

Comment:
```suggestion
```

There aren't references like this in `CONTRIBUTING.md` here, this should be removed.